nemo_rl/data/datasets/response_datasets/openmathinstruct2.py

```
@@ -12,96 +12,59 @@
 
-class OpenMathInstruct2Dataset(RawDataset):
     def __init__(
         self,
+        output_key: str = "expected_answer",
         split: str = "train_1M",
+        split_validation_size: float = 0.05,
         seed: int = 42,
-        test_size: float = 0.05,
-        output_key: str = "expected_answer",
-        prompt_file: Optional[str] = None,
+        **kwargs,
```

Ruff (0.14.13)

Unused method argument: `kwargs`

(ARG002)